pkg/scheduler/framework/plugins/imagelocality/image_locality.go

```diff
@@ -80,14 +80,16 @@ func New(_ runtime.Object, h framework.FrameworkHandle) (framework.Plugin, error
 
 // calculatePriority returns the priority of a node. Given the sumScores of requested images on the node, the node's
 // priority is obtained by scaling the maximum priority value with a ratio proportional to the sumScores.
-func calculatePriority(sumScores int64) int64 {
-	if sumScores < minThreshold {
-		sumScores = minThreshold
-	} else if sumScores > maxThreshold {
-		sumScores = maxThreshold
+func calculatePriority(sumScores int64, sizeContainers int) int64 {
+	minThresholds := minThreshold * int64(sizeContainers)
+	maxThresholds := maxThreshold * int64(sizeContainers)
+	if sumScores < minThresholds {
```

Comment:
Actually, why don't we do this normalization in `sumImageScores`, per image.

Reply:
This is what I thought, but normalization per image means rewrite the whole logic, i.e. redefine the formula used here.

Let me evaluate this approach.

Reply:
done